charts/sentry/templates/relay/_helper-sentry-relay.tpl

```diff
@@ -74,8 +74,27 @@ config.yml: |-
     {{- else }}
     redis: "{{ $redisProto }}://{{ $redisHost }}:{{ $redisPort }}/{{ $redisDb }}"
     {{- end }}
+
+    {{- if ((.Values.kafkaTopicOverrides).prefix) }}
+    topics:
+      metrics_sessions: "{{ default "" .Values.kafkaTopicOverrides.prefix }}ingest-metrics"
+      events: "{{ default "" .Values.kafkaTopicOverrides.prefix }}ingest-attachments"
+      transactions: "{{ default "" .Values.kafkaTopicOverrides.prefix }}ingest-transactions"
+      outcomes: "{{ default "" .Values.kafkaTopicOverrides.prefix }}ingest-outcomes"
+      outcomes_billing: "{{ default "" .Values.kafkaTopicOverrides.prefix }}ingest-outcomes"
+      metrics_generic: "{{ default "" .Values.kafkaTopicOverrides.prefix }}ingest-performance-metrics"
+      profiles: "{{ default "" .Values.kafkaTopicOverrides.prefix }}profiles"
+      replay_events: "{{ default "" .Values.kafkaTopicOverrides.prefix }}ingest-replay-events"
+      replay_recordings: "{{ default "" .Values.kafkaTopicOverrides.prefix }}ingest-replay-recordings"
+      monitors: "{{ default "" .Values.kafkaTopicOverrides.prefix }}ingest-monitors"
+      spans: "{{ default "" .Values.kafkaTopicOverrides.prefix }}snuba-spans"
+      metrics_summaries: "{{ default "" .Values.kafkaTopicOverrides.prefix }}snuba-metrics-summaries"
+      cogs: "{{ default "" .Values.kafkaTopicOverrides.prefix }}shared-resources-usage"
+      feedback: "{{ default "" .Values.kafkaTopicOverrides.prefix }}ingest-feedback-events"
```

Comment:
Could you tell me, what is this topic `ingest-feedback-events`? I don't see the provisioning of this topic [here](https://github.com/sentry-kubernetes/charts/blob/develop/charts/sentry/values.yaml#L2004-L2106)

Reply:
https://github.com/sentry-kubernetes/charts/issues/1359#issuecomment-2284916724 has `ingest-feedback-events`, if it's not needed then it could be removed.

Reply:
https://github.com/getsentry/sentry/blob/f8c66f4440477e2adcc6bf2eedfa56a09f714936/src/sentry/conf/types/kafka_definition.py#L31

Seems like it is in the kafka definition, we may need to provision it?